### PR TITLE
Document how to run JMH benchmarks

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Review by ktoso — APPROVED** [review]
Good :) I thought we had that, weird

File: akka-bench-jmh/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Akka Microbenchmarks
+
+This subproject contains some microbenchmarks excercising key parts of Akka.
+
+You can run them like:
+
+   project akka-bench-jmh
+   jmh:run -i 3 -wi 3 -f 1 .*actor.*
```
Comment (patriknw) [inline akka-bench-jmh/README.md]: I think you almost always run one specific class, so better to write out a full class name in the example.

**Review by patriknw — COMMENTED** [review]
Good, should this go into `CONTRIBUTING` (also)?

**Comment by raboof** [comment]
We could add a link from `CONTRIBUTING.md` to here - though that document is already quite long, not sure if it'd help much...

**Comment by ktoso** [comment]
LGTM, this is good enough

**Pull request merged**